app/models/manageiq/providers/kubevirt/infra_manager/provision/cloning.rb

```diff
@@ -115,7 +107,7 @@ def param_substitution!(object, params)
   # @params object [String] Object on which substitution takes place.
   # @returns [String] The outcome of substitution.
   #
-  def sub_specific_object(params, object)
+  def substitute_string_param(object, params)
```

Comment:
Can we name this with a `!` since it modifies the parameter?  Also, interestingly, it says `result = object`, then later does object.sub!, but takes the return value and puts it back in result.

Reply:
Yeah I'm not sure this was ever working in the case where the param is not a string...working on a spec test for that case

Reply:
Okay done, I feel better changing this method now that it has better spec coverage :laughing: